src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -117,12 +118,21 @@ public KillBillHttpClient(final String kbServerUrl, final String username, final
         this.apiKey = apiKey;
         this.apiSecret = apiSecret;
 
-
         final AsyncHttpClientConfig.Builder cfg = new AsyncHttpClientConfig.Builder();
 
+        if(requestTimeout != null)
```

Comment:
Nit but could you respect the formatting (i.e. `if (requestTimeout != null) {`)?